src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -283,8 +283,6 @@ private void updatePrivilegeInSessionLevel(String sectionName, String sessionNam
     }
     
     /**
-     * used for bulk update of privileges for sectionName
```

Comment:
As it so happens, I found a way for CS to disallow empty description.

But this results in a dilemma:
- Disallowing both empty description and empty `@return` tag necessitates duplication.
- Disallowing empty `@return` tag makes documenting legitimate, non-trivial return values missable.
- Disallowing empty description is against the coding standard for return statements.

For me disallowing empty `@return` is still the best alternative since "coding standard" is not a bible.